tests/dummy/app/templates/route-components/assets.hbs

```diff
@@ -89,6 +89,7 @@
     <IconBox @name="map" />
     <IconBox @name="menu-control" />
     <IconBox @name="menu" />
+    <IconBox @name="mention" />
```

Comment:
If it's supposed to be in alphabetical order - then it's not :)

Reply:
You're right. Damn :D